x-pack/solutions/observability/plugins/slo/server/plugin.ts

```diff
@@ -180,6 +193,8 @@ export class SLOPlugin
       ?.start(plugins.taskManager, internalSoClient, internalEsClient)
       .catch(() => {});
 
+    this.healthTask?.start({ taskManager: plugins.taskManager }).catch(() => {});
```

Comment:
i have a same question here, this needs to be started if user have SLOs, or at least add an easy check around platinum license.

Reply:
👍🏻 I can add that around both task

Reply:
Done in https://github.com/elastic/kibana/pull/209614/commits/c4771728acc0cd24c5d62926187f9fdb3b8ef790

I'll refactor the other task to have a similar check but in a follow-up PR, this one is already too big